ceno_zkvm/src/instructions/riscv/rv32im.rs

```diff
@@ -337,14 +340,10 @@ impl<E: ExtensionField> Rv32imConfig<E> {
             }
         });
 
-        for (insn_kind, (_, records)) in InsnKind::iter()
```

Comment:
(Not introduced by this PR) this block is only for statistics info, so can we make whole block only run when log level <= tracing?

Reply:
In principle, yes.  What are you trying to achieve with that suggested change?

The code here is only executed once, and not part of any inner loop etc; so performance is not an issue.

Reply:
This code is on critical path, especially it go through all the instruction witness, it practice application it will up to 10M or more. We should avoid that and apparently it's just for debug info

Reply:
Ohoh, the result has been grouped by records, got it and there is no issue